ping/ping.c

```diff
@@ -337,6 +344,9 @@ main(int argc, char **argv)
 		.ni.query = -1,
 		.ni.subject_type = -1,
 	};
+
```

Comment:
Why empty line?

Reply:
Readability. iputils 30 years code style si horrible. But sure, I can remove it.